Update node.js.yml

OluwasanmiVic123 opened the pull request.

de-marauder commented on .github/workflows/node.js.yml:
This isn't going to work. Preferably, split up the terraform CI/CD From the application CI/CD. 

The application CI is ok, you just need to separate it from that for terraform.

de-marauder reviewed (changes requested):
This isn't going to work. Preferably, split up the terraform CI/CD From the application CI/CD.

The application CI is ok, you just need to separate it from that for terraform.

OluwasanmiVic123 commented:
Noted.
I will work on it.

Best regards,
Victoria.

On Fri, 9 Jun 2023 at 12:19, De-marauder ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> This isn't going to work. Preferably, split up the terraform CI/CD From
> the application CI/CD.
>
> The application CI is ok, you just need to separate it from that for
> terraform.
> ------------------------------
>
> On .github/workflows/node.js.yml
> <https://github.com/de-marauder/Altschool-Capstone-Project/pull/3#discussion_r1224177043>
> :
>
> This isn't going to work. Preferably, split up the terraform CI/CD From
> the application CI/CD.
>
> The application CI is ok, you just need to separate it from that for
> terraform.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/de-marauder/Altschool-Capstone-Project/pull/3#pullrequestreview-1471752829>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZ5W54O2TKIGK4BO637YUWTXKMBF3ANCNFSM6AAAAAAZAOUQP4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
